tests/test_check_size.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+echo "Running cc-snapshot interface tests..."
+
+fail() {
+  echo "FAIL: $1"
+  exit 1
+}
+
+pass() {
+  echo "PASS: $1"
+}
+
+TEST_SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+CC_SNAPSHOT="${TEST_SCRIPT_DIR}/../cc-snapshot"
+
+if [[ ! -x "${CC_SNAPSHOT}" ]]; then
+    echo "Error: unable to find ${CC_SNAPSHOT} or it is not executable"
+    exit 1
+fi
+
+### Testing check_size and prepare_tarball
+
+TESTING_SKIP_ROOT_CHECK=1 dd if=/dev/zero of=/tmp/fake.tar bs=1M count=10 status=none
```

Comment:
why are we using `TESTING_SKIP_ROOT_CHECK` here? that's only relevant to cc-snapshot, not the dd command